tools/uw-frame-static/build.js

```diff
@@ -18,7 +18,7 @@ mkdirp('uw-frame-static/target/css/themes/', function(error) {
 });
 
 
-const themes = ['uw-madison', 'uw-system', 'uw-river-falls',
+const themes = ['default', 'uw-madison', 'uw-system', 'uw-river-falls',
```

Comment:
I checked how theme preference is stored client-side in session storage: it's by name rather than by array index (hurray!) so this should deploy cleanly even if static theming mattered in prod, which I bet we're not actually doing any of anyway. :smile: